modules/hanna-sprinkles/src/Carousel.tsx

```diff
@@ -0,0 +1,48 @@
+import './_/initHannaNamespace.js';
+
+import React, {ReactNode} from 'react';
+import ReactDOM from 'react-dom';
+import qq from '@hugsmidjan/qj/qq';
+import { Carousel, CarouselProps } from '@reykjavik/hanna-react/Carousel';
+
+import { autoSeenEffectsRefresh, autoSeenEffectWrapperProps } from './_/addSeenEffect.js';
+
+const getCarouselData = (elm: HTMLElement): CarouselProps => {
+  const items = qq<HTMLElement>('.Carousel__itemlist', elm).map(
+    (itemElm: HTMLElement): ReactNode => {
+      const items = Array.from(itemElm.children).map((child) => {
+        const childHTML = child.innerHTML;
+        // eslint-disable-next-line react/jsx-key
+        return <div dangerouslySetInnerHTML={{ __html: childHTML }} />;
+      });
+
+      return items;
+    }
+  );
+
+  const children = items.flat();
```

Comment:
I'm going to make a few changes around here:

* Use `q` (`.querySelector`) instead of `qq` (`.querySelectorAll`) to avoid the extra `.map` closure
* Add a `key={i}` prop and remove the `eslint-disable` direcctive
* ...the above obviating items.flat()